x-pack/solutions/search/plugins/search_query_rules/public/components/query_ruleset_detail/query_rule_draggable_list.tsx

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import {
+  EuiDragDropContext,
+  EuiDroppable,
+  EuiDraggable,
+  EuiPanel,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiText,
+  EuiBadge,
+  EuiTextColor,
+  EuiButtonIcon,
+  useEuiTheme,
+  euiDragDropReorder,
+} from '@elastic/eui';
+import {
+  QueryRulesQueryRule,
+  QueryRulesQueryRuleCriteria,
+} from '@elastic/elasticsearch/lib/api/types';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { i18n } from '@kbn/i18n';
+
+export interface QueryRuleDraggableListProps {
+  rules: QueryRulesQueryRule[];
+  onReorder: (queryRules: QueryRulesQueryRule[]) => void;
+}
+export const QueryRuleDraggableList: React.FC<QueryRuleDraggableListProps> = ({
+  rules,
+  onReorder,
+}) => {
+  const {
+    euiTheme: { colors },
+  } = useEuiTheme();
+  const criteriaDisplay = (criteria: QueryRulesQueryRuleCriteria) => (
+    <EuiText size="s">
+      <EuiBadge>{criteria.metadata}</EuiBadge>&nbsp;
+      <EuiTextColor color={colors.textPrimary}>{criteria.type}</EuiTextColor>&nbsp;
+      {criteria.values?.join(',')}
```

Comment:
should we add a space after the `,` to add a game between terms? just from a visual perspective?
```suggestion
      {criteria.values?.join(', ')}
```